ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.5__4.19.yaml

```
@@ -130,6 +130,58 @@ tests:
           cpu: 1000m
           memory: 512Mi
     workflow: optional-operators-ci-aws
+- always_run: false
+  optional: true
+  as: e2e-test-hcp-aws
```

```suggestion
- always_run: false
  as: e2e-test-hcp-aws
  optional: true
```
I think it will resort the keys like so.